sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -316,6 +316,37 @@ spring:
           useInsecureTrustManager: true
 ```
 
+### Refresh certificates when have un trust certificate
+
+When the inbound certificate is not trusted, the KeyVaultKeyStore can fetch 
+certificates from KeyVault if the following property is configured:
+
+```yaml
+azure:
+  keyvault:
+    jca:
+      refresh-certificates-when-have-un-trust-certificate: true
+```
+
+Note: If you set refresh-certificates-when-have-un-trust-certificate=true, your server will be vulnerable
+to attack, because every untrusted certificate will cause your application to send a re-acquire certificate request.
+
+You can also manually refresh the certificate by calling this method:
+```java
+KeyVaultCertificates.refreshCertsInfo();
+```
```

Comment:
Move this after `### Refresh certificate periodically`. And add header `### Refresh certificate by java code`